firebase-firestore/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Unreleased
+- [changed] Changed the internal handling for locally updated documents that
```

Comment:
Hm. This tells me that there's a "slight behavior change" but doesn't say what that is... (though does say it may affect the metadata flag). But I see that's what we've got for the web port. Is this worth expanding on at all?

Reply:
It's pretty much impossible to put into words, so we decided to be vague about it.